game.c

```diff
@@ -208,23 +208,30 @@ void init(void) {
 	cam.pitch_deg = 0.0f;
 	cam.yaw_deg = 0.0f;
 	cam.turn_vel = HMM_Vec2(0.0f, 0.0f);
+
+	sapp_lock_mouse(false);
```

Comment:
isn't this basically a no-op? by default, the mouse isn't locked, so it's probably best to omit explicitly unlocking the mouse when the game starts up?